webapi/Models/Storage/CopilotChatMessage.cs

```diff
@@ -14,7 +14,7 @@ namespace CopilotChat.WebApi.Models.Storage;
 /// <summary>
 /// Information about a single chat message.
 /// </summary>
-public class ChatMessage : IStorageEntity
+public class CopilotChatMessage : IStorageEntity
```

Comment:
Any reason we are chaging this?

Reply:
It clashes with Microsoft.SemanticKernel.AI.ChatCompletion.ChatCompletion